plugin/src/test/kotlin/spp/jetbrains/VariableParserTest.kt

```diff
@@ -0,0 +1,42 @@
+package spp.jetbrains
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import spp.jetbrains.sourcemarker.VariableParser
+import java.util.ArrayList;
+import java.util.regex.Pattern
+
+internal class VariableParserTest {
+
+    var variablePattern =
+        Pattern.compile("(\\\$tt|\\\$deleteDate|\\\$executorService|\\\$FIVE_MINUTES_MILLIS|\\\$mapTodos|\\\$GET|\\\$HEAD|\\\$POST|\\\$PUT|\\\$PATCH|\\\$DELETE|\\\$OPTIONS|\\\$TRACE)(?:\\s|$)")
+    var templateVariablePattern =
```

Comment:
Don't hardcode the variable pattern. This should be done by VariableParser.

Each test should define vars and pattern. Something like:
```kotlin
val contextVars = listOf("deleteDate", "executorService")
val pattern = "hello$deletedDate ${executorService}"
```